.github/workflows/license_finder.yml

```diff
@@ -13,8 +13,7 @@ jobs:
     timeout-minutes: 30
 
     steps:
-      - name: Check out code in rdk directory
-        uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

Comment:
Did you mean to take out the name here?

Reply:
yup -- `name` is required for `run` steps, but optional for `uses` steps. can add back if you have a preference